src/components/global/Footer.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+function Footer() {
+  return (
+    <footer
+      style={{ bottom: 0 }}
+      className="relative flex flex-col h-44 justify-center align-items-center w-full p-5 z-50 bg-regal-blue-dark"
+    >
+      <div className="text-grey-700">
+        Made with
+        <span className="text-cyan-700 font-bold ml-1 mr-1">&lt;/&gt; </span> by
+        <a
```

Comment:
Avoid using `<a>` tag in react , use can use `<Link>` tag

Reply:
oops! , forgot to change them, will change it to link